### ISSUE TITLE
Add captcha to sign-up page

### ISSUE THREAD

**Issue by junyada100** [VaultExpress/vault-express#60]

**Comment by xarrijorge** [comment]
Hey, should we use Google reCaptcha for this? 
https://www.google.com/recaptcha/intro/v3beta.html

**Comment by junyada100** [comment]
yes, it think it is an easy way to solve this issue, anyway, last time i tried it, it needs us to allow inline-style for content security policy which actually i tried to disallow that.

but it is common for most web app then we might compromise that ^ ^
